lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -457,8 +457,7 @@ def success_from(action, response)
           refund_status = response.dig('status') || response.dig('payment', 'status')
           %w(REFUNDED REFUND_REQUESTED).include?(refund_status)
         else
-          response['status'] != 'REJECTED' # default prior to revision, should be removed after building out the rest
-        end
```

Comment:
Did you mean to remove this `end` statement?

Reply:
lol that was the issue hah